frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -390,7 +390,11 @@ export function createYAxisOptions({
             const { total_count } = deserializeCellTypeMetadata(
               value as CellTypeMetadata
             );
-            return total_count > 10000 ? ">10k" : `${total_count}`;
+            let formattedCount = `${total_count}`;
+            if (total_count > 10000) {
+              formattedCount = `${Math.round(total_count / 1000)}k`;
+            }
+            return formattedCount;
```

Comment:
const formattedString = Intl.NumberFormat("en-US", {
      maximumFractionDigits: 1,
      notation: "compact",
    }).format(n);
    return `${formattedString}${formattedString !== n.toString() ? "+" : ""}`;
    
    for when we hit 1M of a single cell type 🚀, will also give us  `x.xK`

Reply:
Oh right, will fix!